hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,6 @@
+RELEASE_TYPE: patch
+
+This release adds the strategy :func:`~hypothesis.extra.numpy.mutually_broadcastable_shapes`,
+which generates multiple array shapes that are mutually broadcast-compatible with a
+user-specified base-shape. This is a generalization of
```

Comment:
'with an *optional* user-specified base shape'?  Also nice to call out some applications of such a strategy, and maybe link to `:issue:'1969'`.